src/components/commons/Footer.tsx

```diff
@@ -1,20 +1,93 @@
-import { Box, Container, Flex, Text } from '@chakra-ui/react'
-import { FC } from 'react'
-import { Link } from 'react-router-dom'
+import { Box, Container, Flex, HStack, Image, Link, Text, useColorModeValue, VStack } from '@chakra-ui/react'
+import { FC, MouseEvent } from 'react'
+import { FaEnvelope, FaFacebook, FaGithub, FaGlobe, FaHeart, FaInstagram } from 'react-icons/fa'
+import { Link as RRDLink } from 'react-router-dom'
+import customTheme from '../../assets/theme'
 import { PATHS } from '../../util/paths'
 
-export const Footer: FC = () => (
-  <Box as="footer">
-    <Container py={8} as={Flex} justifyContent="space-between" direction={{ base: 'column', sm: 'row' }} maxW="6xl">
-      <Flex mb={{ base: 4, sm: 0 }} justifyContent={{ base: 'center', sm: 'flex-start' }}>
-        {/** todo: Enter content */}
-        <Link to={PATHS.IMPRESSUM}>Impressum</Link>
-      </Flex>
-      <Flex direction="column" justifyContent={{ base: 'center', sm: 'flex-end' }}>
-        <Text mt={2} textAlign={{ base: 'center', sm: 'right' }}>
-          &copy; {new Date().getFullYear()} • Kir-Dev
-        </Text>
-      </Flex>
-    </Container>
-  </Box>
-)
+export const Footer: FC = () => {
+  const kirDevLogo = useColorModeValue('/img/kirdev.svg', '/img/kirdev-white.svg')
+  const hkLogo = useColorModeValue('/img/hk.svg', '/img/hk-white.svg')
```

Comment:
WHy aren't you manipulating the SVG itself with inline colors or css?

Reply:
this was easier 😄

Reply:
I'm not really in the mood for editing the SVGs, so if it's okay with you, I wouldn't change this

Reply:
Okay OFC